### ISSUE TITLE
Policy Loss sometimes value equal zero(0)

### ISSUE THREAD

**Issue by zhangquanwei962** [NVlabs/DiffusionNFT#28]
Hi, Is my policy loss = 0 because there's no difference in my within-group averages? I'm currently training with 48 samples per epoch, using reward from MLLM, learning rate of 0.0003, and train.beta = 0.0001.

**Additionally, does this training curve look normal?**

<img width="1786" height="629" alt="Image" src="https://github.com/user-attachments/assets/b650efbe-ca3a-4599-8f81-d14bde13a77d" />

<img width="1781" height="624" alt="Image" src="https://github.com/user-attachments/assets/d7034b3a-c6b4-4f58-9d2a-6dfa5087b86a" />

<img width="1775" height="625" alt="Image" src="https://github.com/user-attachments/assets/726022c3-13dc-4187-a3b2-cad65a960eeb" />

<img width="1779" height="626" alt="Image" src="https://github.com/user-attachments/assets/a484a664-c09d-417d-b00b-c9f46199bd43" />

<img width="1786" height="615" alt="Image" src="https://github.com/user-attachments/assets/97e420a0-ca6a-4fc7-b669-50d07bb49fd5" />

**Comment by FaltingsA** [comment]
@zhangquanwei962  I guess the the valid_mask of the mini-batch has all False values, that means  all samples in the mini-batch are banned;
